src/System.Management.Automation/FormatAndOutput/out-console/ConsoleLineOutput.cs

```diff
@@ -87,8 +87,15 @@ internal DisplayCellsPSHost(PSHostRawUserInterface rawUserInterface)
 
         internal override int Length(string str, int offset)
         {
-            Dbg.Assert(offset >= 0, "offset >= 0");
-            Dbg.Assert(string.IsNullOrEmpty(str) || (offset < str.Length), "offset < str.Length");
+            if (string.IsNullOrEmpty(str))
```

Comment:
@iSazonov Why are changes to this method necessary to fix the issue? In particular do we need replace assert with exception in  this internal method?

Reply:
The change is not direct fix for the issue. It is related the issue. It makes the code more reliable.
If we don't change the method we will have different behavior for debug and release code that might confuse.